api/services/locationService.ts

```diff
@@ -1,18 +1,21 @@
 import { uniq } from 'lodash';
 import { makeSCOSRequest } from './scosService';
 import getLogger from '../utils/logger';
-import { AGENCIES_TABLE, CATEGORIES_TABLE } from '../utils/constants';
+import { AGENCIES_TABLE, CATEGORIES_TABLE } 
+  from '../utils/constants';
 import LocationDto from '../models/dto/locationDto';
 import ScosAgencyDto from '../models/scosApi/scosAgencyDto';
 import { notEmpty } from '../utils/typeGuards';
+import { getFTLocationData } from './freshtrakService';
+import freshtrakLocationDto from '../models/freshtrakAPI/freshtrakLocationDto';
 
 const log = getLogger('locationService');
 
-function mapToLocationDto(agency: ScosAgencyDto): LocationDto | null {
+async function mapToLocationDto(agency: ScosAgencyDto): Promise<LocationDto | any> {
```

Comment:
```suggestion
async function mapToLocationDto(agency: ScosAgencyDto): Promise<LocationDto> | null {
```

Reply:
Great catch!  I had noticed that didn't work earlier, but didn't realize that FreshtrakData was FreshTrakData in the Dto, so I kludged it with <any>.